### ISSUE TITLE
NanoPlot crash

### ISSUE THREAD

**Issue by arpee51** [wdecoster/NanoPlot#98]
Hi Wouter,
Thanks so much for creating NanoPlot.
Used it 3 weeks ago without issues. Today it crashed (see log).
The only difference I can see is that this time, the Nanopore data were generated with barcodes.
Thanks for your help.
Best wishes,
Armand

[NanoPlot_20180926_0913.log](https://github.com/wdecoster/NanoPlot/files/2418531/NanoPlot_20180926_0913.log)

**Comment by wdecoster** [comment]
Hi Armand,

I've never seen this issue before, but googling brought me to this thread: https://github.com/facebook/prophet/issues/256

It suggests your pandas and/or numpy modules are to blame. Could you try updating these and see if that changes anything?

`pip install pandas numpy -U`

**Comment by arpee51** [comment]
Thanks, I've updated pandas & numpy in virtualenv (pip3) and got warnings about matplotlib so I also updated matplotlib. I still got an error (unknown property height) but my plots are nonetheless in the NanoPlot output folder. 
I found the same error in issue #82  and after updating seaborn, the problem was solved. 
(All that remains are several "MatplotlibDeprecationWarning"s).
Solved - Thanks very much.
